backend/src/main/java/ai/verta/modeldb/entities/versioning/RepositoryEntity.java

```diff
@@ -184,14 +184,17 @@ public void update(SetRepository request) throws InvalidProtocolBufferException
     final Repository repository = request.getRepository();
     this.name = repository.getName();
     this.description = repository.getDescription();
-    this.date_updated = new Date().getTime();
     this.repository_visibility = repository.getRepositoryVisibilityValue();
     this.repositoryAccessModifier = repository.getRepositoryAccessModifierValue();
     this.workspace_id = repository.getWorkspaceId();
     this.workspace_type = repository.getWorkspaceTypeValue();
     updateAttribute(repository.getAttributesList());
   }
 
+  public void update() {
+    this.date_updated = new Date().getTime();
```

Comment:
will this be set whenever the object is updated/ saved?

Reply:
added it into update that is called from setRepository